_maps/RandomRuins/LavaRuins/lavaland_surface_dorffort.dmm

```diff
@@ -0,0 +1,57 @@
+"a" = (/turf/template_noop,/area/template_noop)
+"b" = (/turf/closed/mineral/ash_rock,/area/mine/explored)
```

Comment:
I've mentioned it before but interference with the lavaland terrain generator is putting holes in the ruin additionally. It may be a case we will have to use a solid wall instead.